### ISSUE TITLE
Do we have an option to configure 24 hpur format?

### ISSUE THREAD

**Issue by AlbyJoseph** [owsolutions/amazing-time-picker#65]
I need to set the time picker format (12 or 24 hour format) using a configuration. Is there any option available?

**Comment by torabian** [comment]
Unfortunately, this option is not available now. I am going to make a card for this to create, seems requests are so high.

**Comment by alan345** [comment]
Hi! Very interested also! thanks @torabian for your work! Very useful

**Comment by kovalchukim** [comment]
Also interested. Thanks @torabian for amazing time-picker!

**Comment by silentsnooc** [comment]
I'd love to see that too.

**Comment by mtuzinskiy** [comment]
That's a deal breaker for our app :( I love the lib though, seems like it's the most mature one out there currently, but the absence of this feature just ruins everything

**Comment by johnayling** [comment]
The display of the time in the input field doesn't make sense to have 24 hour format also with AM or PM identifier.

eg. 08:30 is 8:30 AM and 20:30 is 8:30 PM

Whereas the output values of 08:30 AM and 08:30 PM mixes 12hr and 24hr formats.

Suggest format option that displays 8:30 AM / 8:30 PM for 12hr format OR 08:30 / 20:30 for 24hr format

**Comment by saiar** [comment]
Hi @torabian, Thanks for the beautiful library! Just wanted to add a +1 to this feature request.

**Comment by Chris2011** [comment]
Yes that would be great. Here is the proposal, how it looks like in android apps: https://i.stack.imgur.com/a1yJn.png

**Comment by torabian** [comment]
I am looking forward to do cover this during this weekend. Will be available from 1.7.0

**Comment by saiar** [comment]
@torabian Great! Thanks for taking this up!

**Comment by ramonvic** [comment]
@torabian any news of this feature?

**Comment by cmddavid** [comment]
Waiting for this feature as well, this time picker is amazing but in a lot of European countries we use the 24h clock. I implemented the time picker in one of our tools, but not having the ability to switch between 12h clock and 24h clock could be a reason for my superiors to reject the time picker(even though I love it :D).

**Comment by Vladimex** [comment]
Is there any news about the 24h feature?

**Comment by 2taz** [comment]
Hey guys, when will a 24h feature be available?

**Comment by darionato** [comment]
I need 24h feature too, hope it will'be added soon! Thanks.

**Comment by plvice** [comment]
Hello there, is there any chance to support 24h? Europe here

**Comment by newprogrammer1991** [comment]
Please adde this feature

**Comment by rever96** [comment]
That's the more version-friendly module for time-picker, Would be amazing have 24h feature.

**Comment by osbeug001** [comment]
There is a fork https://github.com/casni14/amazing-time-picker with 24hrs that seems to work.
Maybe it can be merged?

**Comment by Chris2011** [comment]
@osbeug001 you can create a PR with the commits to add this :) or the user who forked it.
